linera-core/src/unit_tests/worker_tests.rs

```diff
@@ -3194,7 +3151,12 @@ async fn test_cross_chain_helper() -> anyhow::Result<()> {
             None,
             bundles012.clone()
         )?,
-        Vec::from_iter(bundles1.iter().cloned().chain(bundles2.iter().cloned()))
+        bundles1
+            .iter()
+            .cloned()
+            .chain(bundles2.iter().cloned())
+            .map(|(_, bundle)| bundle.clone())
```

Comment:
Is the additional clone needed?

```suggestion
            .map(|(_, bundle)| bundle)
```